docs/guides/using-beluga-amcl.md

```diff
@@ -1,3 +1,153 @@
-# Using Beluga AMCL on an AMR
+# Using Beluga AMCL
 
-Coming soon!
+## Prerequisites
+
+Before we jump in, let’s make sure you’ve got everything you need:
+
+- **Hardware Requirements:**
+  - **Processor:** a dual-core x86-64 CPU should be enough to handle real-time localization.
+  - **Memory:** at least 2 GB of RAM to comfortably run ROS 2 and AMCL, plus other tasks.
+  - **Storage:** ensure you have around 4 GB of free storage for ROS 2 and mapping data.
+  - **Sensors:**
+    - **2D LiDAR:** scans are essential for localization as Beluga AMCL uses them to match against a known map.
+    - **Odometry:** some ego-motion estimation is also necessary (e.g. wheel encoder-based odometry).
+
+- **Software Requirements:**
+  - **OS:** you can use either Ubuntu 22.04 LTS (Jammy Jellyfish) or Ubuntu 20.04 LTS (Focal Fossa).
+  - **ROS 2:** you can use either ROS 2 Humble Hawksbill or ROS 2 Jazzy Jalisco.
+
+:::{important}
+Most localization and SLAM packages in ROS 2 that take 2D LiDAR scans expect them published as `sensor_msgs/LaserScan` messages over the `/scan` topic. Likewise, these packages also expect odometry broadcasted as `odom` to `base_link` transforms over `tf`. That is the case for Beluga AMCL as well. Make sure your system follows this convention before using Beluga AMCL, or take the necessary steps to [reconfigure Beluga AMCL](../packages/beluga_amcl/docs/ros2-reference.md#parameters) to match your own conventions.
+:::
+
+## Setting things up
+
+1. **Install Beluga** by following [its installation guide](../getting-started/installation).
+
+2. **Install additional packages** like [SLAM Toolbox](https://github.com/SteveMacenski/slam_toolbox) and the [Nav2 `map_server`](https://github.com/ros-navigation/navigation2/tree/main/nav2_map_server):
+
+     ```bash
+     sudo apt install ros-${ROS_DISTRO}-slam-toolbox ros-${ROS_DISTRO}-map-server
+     ```
+
+   You will need these for mapping work.
+
+:::{tip}
+Remember to source your ROS 2 environment to use it:
+
+```bash
+source /opt/ros/${ROS_DISTRO}/setup.bash
+```
+:::
+
+## Mapping the environment
+
+Before the robot can localize using Beluga AMCL, we need a map of the environment. Here's how you can create one:
```

Comment:
Some parts of the text use `’` and others `'`, like here in `Here's`.

Reply:
Good catch! Fixed in https://github.com/Ekumen-OS/beluga/pull/447.